### DISCUSSION TITLE
Javascript is blocked... I need to get my history removed or flooded help

### DISCUSSION THREAD

**Discussion by OrangeThunderFire** [3kh0/ext-remover#973] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
HELP

**Comment by AshtonDavies** [comment]
Why can't you clear your history?

**Comment by OrangeThunderFire** [comment]
The school won't let me

**Reply by AshtonDavies** [reply]
> The school won't let me
> 
> On Tue, Mar 26, 2024 at 7:30 PM Ashton Davies ***@***.***>
> wrote:
> 
> > Why can't you clear your history?
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/3kh0/ext-remover/discussions/973#discussioncomment-8922547>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/BERVT66PMDU3KNNT2QXWW6DY2IVNHAVCNFSM6AAAAABFKBETKCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSMRSGU2DO>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
> 

If chrome://policy is not blocked for you, try checking what URLs are whitelisted.

You may be able to visit chrome://settings/clearBrowserData to clear all history and use settings.

**Comment by OrangeThunderFire** [comment]
its not like that. The button to delete history is grey and unclickable

**Reply by Blobby-Boi** [reply]
Just flood it

**Comment by OrangeThunderFire** [comment]
how? Javascript is blocked

**Reply by AshtonDavies** [reply]
There is no reason why you would need a bookmarklet to do something that does not require any specific page or domain, so use your brain.. I mean, a website to flood history.

**Comment by OrangeThunderFire** [comment]
what website?

**Reply by AshtonDavies** [reply]
> what website?
> 
> On Wed, Mar 27, 2024 at 12:42 PM Ashton Davies ***@***.***>
> wrote:
> 
> > Use a website
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/3kh0/ext-remover/discussions/973#discussioncomment-8932911>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/BERVT6ZH7XV2A3NNRIBY7D3Y2MOLBAVCNFSM6AAAAABFKBETKCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSMZSHEYTC>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
> 

If you can't find one, just run the script somewhere or create an HTML page that runs whatever JavaScript flooder you would use. It's not very hard.

**Comment by OrangeThunderFire** [comment]
ur site doesn't work

**Comment by OrangeThunderFire** [comment]
ill check it out

**Comment by OrangeThunderFire** [comment]
It says I need permission

**Comment by OrangeThunderFire** [comment]
@Blobby-Boi @AshtonDavies

**Reply by Blobby-Boi** [reply]
Uhhh idk why I made it so anyone in the world can see

**Comment by OrangeThunderFire** [comment]
[image: image.png]

**Reply by Blobby-Boi** [reply]
Just use any website that can run JavaScript and put the flooder bookmarklet on it

**Comment by OrangeThunderFire** [comment]
can you send me a website? Ashton davies website doesn't work.

**Reply by Blobby-Boi** [reply]
U can try a website that I made a while back ago [too](https://blobby-boi.github.io/BlobeBM/)

**Comment by OrangeThunderFire** [comment]
actually it does, but it makes me flood his website, which would look
pretty suspicious if i had thousands of "bookmark executor" tabs on my
history. @Ashton Davies can you tell me how to use the javascript runner on
other tabs

**Reply by Blobby-Boi** [reply]
That’s what the google lens exploit is for
It doesn’t work yet tho

**Comment by OrangeThunderFire** [comment]
if anyone knows how to code, could you make the following history flooder
so that it floods google classroom instead of the page you are on?

javascript:var num=prompt("How Times Do You Want This Page To Show Up In
your History?\Developed By:  Schoolcheats on
 YT");done=false;x=window.location.href;for (var i=1; i<=num;
i++){history.pushState(0, 0,
i==num?x:i.toString());if(i==num){done=true}}if(done===true){alert("Flooding
Successful!\n "+window.location.href+" \nIs Now In Your History
"+num+(num==1?" time.":" Times. \nDeveloped By: Schoolcheats on  YT"))}

**Reply by Blobby-Boi** [reply]
Uhh I don’t think it’s possible to do that unless you just made it spam windows

**Comment by OrangeThunderFire** [comment]
i alr tried that it doesnt work at all. If you saw my earlier message,
could you try to do that if you know how to code javascript?

**Reply by Blobby-Boi** [reply]
I don’t think it would be possible without spamming windows or smth

**Reply by Blobby-Boi** [reply]
But idk since I suck at coding

**Reply by AshtonDavies** [reply]
One thing to consider is that history flooders do exist, so get that in your head because that means it's possible.

Secondly, JavaScript can indeed spam tabs if you enable popups.

**Comment by OrangeThunderFire** [comment]
k

**Comment by OrangeThunderFire** [comment]
Ashton Davies, thank goodness you are here. I can now use javascript thanks
to your site, but the issue is that it floods the bookmarklet excutor into
my history, and I do not want that site to show up. Doo you happen to know
how to code javascript so you can use one of the original history flooders
and change the code so it floods an educational website say google
classroom? I tried making chatgpt do it but its code didn't work. this is
the code chatgpt provide me with

var num = prompt("How many times do you want Google Classroom to show up in
your history?"); var googleClassroomURL = "https://classroom.google.com/";

for (var i = 1; i <= num; i++) { history.pushState({ page: i }, "Google
Classroom", googleClassroomURL); }

alert("Flooding Successful!\nGoogle Classroom URL - " + googleClassroomURL
+ "\nIs now in your history " + num + (num == 1 ? " time." : " times."));

**Reply by AshtonDavies** [reply, marked as answer]
Here's a history flooder bookmarklet I just made: `javascript:count = prompt("Flood Tab Count"); if (count !== null) {for (var i = 1; i <= count; i++){win = window.open("https://www.youtube.com/watch?v=dQw4w9WgXcQ%22");}}`

**Make sure to enable pop-ups for it to fully work.**

**Comment by OrangeThunderFire** [comment]
nothing happens when I run it

**Reply by Blobby-Boi** [reply]
Ask it again?

**Comment by OrangeThunderFire** [comment]
I tried. Many times.

**Reply by Blobby-Boi** [reply]
javascript:count = website = prompt("URL to flood with (Include https://)"); prompt("Flood Tab Count"); if (count !== null) {for (var i = 1; i<= count; i++) {win = window.open(website);}}
